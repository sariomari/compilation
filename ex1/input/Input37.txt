5664
/*
*/516
wU373//jS((}49
/*]!]?/;9}*///
4757
		7195
)(/*
!*/"EIlkqRr""""aHPyb"L
)26
	"d"""(96
"dB"766
)
8
)900
 2
	/*?*/"Hf"790
//]}m.(?}
6
//"dy"8120
2
)"Jz"464
)/*9
?;}z2q)(]*/894
1
6
5

4855
/*)ss}??)!]]LM9)]}.*///735
//!;5K])Nq)
 fv((("w"(2188
"rt" 686
	"Li")//
)(617
)( 575
140
50
5320
0
1308
3108
3892
/*(}}xD*/	g19
//?s
""7831
4
 /**/79
7086
/*?r(!!;};
!)*/1
ds"N" 920

XI9A2384

"")/*2;?*/6983
"Gf"5305
0
434
406
//(?
//}!//]z!}}6?/(????}Z(GA;}?;}S 4

//}
5
)	6
1
)972
//;?}(}!A
//}bs};A]
""9
//
/*;9;V(])*//*}]??iR?}()

?G(
]]}*/UM9433
8
)oE24
 37
3647
//.])R])}0T
9636
)91
 //(9?1}!??2B 913
3
("U" 7
)))	784
"dnyY"4
(C1SUvwCXf9w4GK//n?;{);6]G!?731
(1963
4
(//;""E)vAaC35/*a;;)]?Q)}
k*///
/*/??}6L}v*/(4
)
6959
(23
880
8
"GtR""")/*/
*/
)"BKVnGao"6912
(21
)	""7
)46
/*kuo?]!))]z);?*/o930
311
ZP41
x5631
"""R"))"IU"//;//(
ZJ89792
(663
26
a3
N2()//](H
(63
/*]qYN*/	24
/*
;/}]6}w?
?]]*/561
505
 7359
"zN")237
339

"QjNjEA"l(tl8082
"mDXw"	
/*N)
*/(5244
L""//t}?)](/]9/?5217
16
	187
	/*/;h]]j;?)H}]}(1;?!
f!;*/(5
89
(10
pI//?rQ
("zw"382
sgWum)/*o}}

()p*/8
0
7949
"t")76
	82
 cNlj"Rx"	(r2
621
/*a*/"Q"523
8515
/*)7Y2g;(}!}(]!3]pK?
*/ KGtmY5L""7
//S(!		/*Y

j*/	o2
15
(721
) ""8193
7
0
/*
*/
//?!r
)7518
	970
9017
)	)
( 75
/*!!7};
]!?
1??!)
18
;X?}]*///!
17
	0
	220
//)//
906
DG11
2
	)b32
22
""/*?2Za!}(Gvr?QxC0;]?;)}k*/22
""KTy"JeT"
7
75
/*3m/?*///qI;}zX!;?)).1296
372
( /*}*//*!
?j!}}(]A?(Tb??!
)!
!
;m*/4
"bPv""dej"44
 "THmMCx"72
 //??!/*R];}
}6?)!gk9;I}?5b.*/589
(""(518
6
353
137
//45b!T
	(18
)/**/6385
(0
1082
81
89
H/*!*/1811
(yp)	"H"
"e"9
(6
928
389
)(j4LT3R8
5
9
)	//R
39
339
/*m(M;.7*/6735
  2518
("y""z"(1348
"X"1692
byAzB(//})y}
/*v)
*/9
(( 9
	21
Y8a746
99
(9
//2
//
)12
5
//4)]i]
16
)B68//?})})(""74
7
)1530
//?}?
/*;;?;!}2?7;!j}mb3!]]}0]o}}?B
!}?]]??6O;U6}wpDe*/91
7807
/*!Bn!;;
w}*/)(8
7535
)6

7776
	/*]6(?*/ 96
906
5
4284
 F27)154
"qz")"T""oD" "pj""QMe"(r2Fp63
/*}8)!
?}}u*///]z(!??o;!I6Es!)!(M!]]]]]PS;5Nl;)?}]}V}]5U""902
) 3700
((5
(97
//;()	0
 //z?
UWQ	""8003
/*6;PYQ}
]
*/99
964
5
/*]!?}(?*/
"eRlu"(416
8109
( //!}]
"")(()8866
Tpr81
//80
(9493
21
272
//l?m}41]6
94
(
3
//
)6335

979
6776
8803
5
)4
m7
SB752
66
/*}/(k)}]!]?;];u
*/)"WH"m2z"Uew"3018
(F6d1
	40
I"yLoypLq"849
d8
245
M5966
dP
 /*!Q;;!
46!6;[;0?s)!!aL]];*/)283
/*}p?)!!(](}!]}?*/)69
)0
/*!*//*]}?*/XT/**/)/*
!B
*///;]u]Un?](/*(8.);}?;4)!}*/"Z"7030
943
35
30
dghxHce8DTQ58
3
()) ywxP34
0
iLZ42y"k"()((6820
 )6655
C	76
67
271
/*}r

]0T]
}]?
}*/(//?u;eg]
//!)(}]?/*!;4
}r?F?
*/m (/*]Wv?!D[}V;64]z]E]fV??d;*/(
"X"I8/*;!


(*/6
 )79
j7857
80
3605
)92
9574
5
7363
/*C?MY*/)Kg
747
//
	9358
)//)]
0
(""73
//t(}m?W()399
)"Iy"2851
//6}(]
)2
E6
9
8106
25
	17
7455
)/*]]
3}]b?};

?Y*/ 	/*3(}c*///)"")56
790
p10"J"761
h9Z007

/*;0bU
(3]l.kt(?)]??];P?0]
!]47;!;!
}}]*/"hFLifIAT"356
994
0

8
//1554
(/*M*/()276
fY67
(0
98
)(588

////2Zys9)//
317
//w]!;]/!"tmHV" /*}]N}l]
zXw;*///!}](?]!]?}
8

)9
)2010
848
9
4
"YAVYT"(//
35
AU /*?]}*/qv0J//]
548
///
///*)
dH?[;??
;?U*///?/*!b)]}!))*/7
JOEm9J/*}
;b
(?
}?};(O?;)H!!29*///?)!(0?;w
/*]8}1;).
N?}]*/)
1237
J//]yYU9435
83
/*;]
*/B"")7579
 //"j"(8
)737
"CXUDcI"3
"S"10
)""815
/*;;}v;;/e)]3][*///)87
(13
29
844
/*???*//*;!
*/ 684
4
1
Q	G7106
3941
//
383
/*g]V}s?z*/	//A
"K"9215
)0
B/*;!*/""//(;(!jWc
(jwx(((/*Y{5)*/(/*
/;]H*/))(//
//)w!M?7{)!
//
"iQ"N8D7392
t6l9Z/*]9(}!z]?
)o?]?My!E];(o;}?
A{s];}bI;(c)6)4
!;*/)rcrmEZK32822
9210
//
7218
( 37
 1
(46
)
 //?;(
2
z)(77
/*;!](*/
3242
(//(Gn9M/*
?Z1*/(//}jG;A.( /*n!T;!!}7]V3*/103
//
1599
//e;?2
 	15
(//!(//};Ci(t;)
//"d"7682

5057
""/*;
K?;?t1];*/2
505
837
326
0

53
)24
9631
()
5
9931
)""()""
262
DE729S3(//CD?;;(;/*;}?}?}/?;
;H1/]]())p]?u.n};6}])/Q*/454
Ww2e)