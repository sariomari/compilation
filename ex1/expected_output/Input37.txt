INT(5664)[1,1]
INT(516)[3,3]
ID(wU373)[4,1]
INT(49)[6,1]
INT(4757)[9,1]
INT(7195)[10,3]
RPAREN[11,1]
LPAREN[11,2]
STRING("EIlkqRr")[12,4]
STRING("")[12,13]
STRING("aHPyb")[12,15]
ID(L)[12,22]
RPAREN[13,1]
INT(26)[13,2]
STRING("d")[14,2]
STRING("")[15,1]
LPAREN[15,3]
INT(96)[15,4]
STRING("dB")[16,1]
INT(766)[16,5]
RPAREN[17,1]
INT(8)[18,1]
RPAREN[19,1]
INT(900)[19,2]
INT(2)[20,2]
STRING("Hf")[21,7]
INT(790)[21,11]
INT(6)[23,1]
STRING("dy")[25,1]
INT(8120)[25,5]
INT(2)[27,1]
RPAREN[28,1]
STRING("Jz")[28,2]
INT(464)[28,6]
RPAREN[29,1]
INT(894)[30,12]
INT(1)[31,1]
INT(6)[32,1]
INT(5)[33,1]
INT(4855)[35,1]
INT(735)[39,1]
ID(fv)[41,2]
LPAREN[42,1]
LPAREN[42,2]
LPAREN[42,3]
STRING("w")[42,4]
LPAREN[42,7]
INT(2188)[42,8]
STRING("rt")[43,1]
INT(686)[43,6]
STRING("Li")[44,2]
RPAREN[44,6]
RPAREN[45,1]
LPAREN[45,2]
INT(617)[46,1]
RPAREN[47,1]
LPAREN[47,2]
INT(575)[47,4]
INT(140)[48,1]
INT(50)[49,1]
INT(5320)[50,1]
INT(0)[51,1]
INT(1308)[52,1]
INT(3108)[53,1]
INT(3892)[54,1]
ID(g19)[55,11]
STRING("")[57,1]
INT(7831)[57,3]
INT(4)[58,1]
INT(79)[60,5]
INT(7086)[61,1]
INT(1)[64,4]
ID(ds)[65,1]
STRING("N")[65,3]
INT(920)[65,7]
ID(XI9A2384)[67,1]
STRING("")[69,1]
RPAREN[69,3]
INT(6983)[69,11]
STRING("Gf")[70,1]
INT(5305)[70,5]
INT(0)[71,1]
INT(434)[72,1]
INT(406)[73,1]
ID(S)[77,1]
INT(4)[77,3]
INT(5)[80,1]
RPAREN[81,1]
INT(6)[81,3]
INT(1)[82,1]
RPAREN[83,1]
INT(972)[83,2]
STRING("")[86,1]
INT(9)[86,3]
ID(UM9433)[94,6]
INT(8)[95,1]
RPAREN[96,1]
ID(oE24)[97,1]
INT(37)[99,2]
INT(3647)[101,1]
INT(9636)[103,1]
RPAREN[104,1]
INT(91)[104,2]
INT(913)[106,2]
INT(3)[107,1]
LPAREN[108,1]
STRING("U")[108,2]
INT(7)[108,6]
RPAREN[109,1]
RPAREN[109,2]
RPAREN[109,3]
INT(784)[109,5]
STRING("dnyY")[110,1]
INT(4)[110,7]
LPAREN[111,1]
ID(C1SUvw)[111,2]
ID(CXf9w4GK)[112,1]
INT(731)[113,1]
LPAREN[114,1]
INT(1963)[114,2]
INT(4)[115,1]
LPAREN[116,1]
STRING("")[118,1]
ID(E)[118,3]
RPAREN[118,4]
ID(vAaC35)[118,5]
LPAREN[123,5]
INT(4)[123,6]
RPAREN[124,1]
INT(6959)[125,1]
LPAREN[126,1]
INT(23)[126,2]
INT(880)[127,1]
INT(8)[128,1]
STRING("GtR")[129,1]
STRING("")[129,6]
RPAREN[129,8]
RPAREN[131,1]
STRING("BKVnGao")[131,2]
INT(6912)[131,11]
LPAREN[132,1]
INT(21)[132,2]
RPAREN[133,1]
STRING("")[133,3]
INT(7)[133,5]
RPAREN[134,1]
INT(46)[134,2]
ID(o930)[136,16]
INT(311)[137,1]
ID(ZP41)[138,1]
ID(x5631)[139,1]
STRING("")[140,1]
STRING("R")[140,3]
RPAREN[140,6]
RPAREN[140,7]
STRING("IU")[140,8]
ID(ZJ89792)[142,1]
LPAREN[143,1]
INT(663)[143,2]
INT(26)[144,1]
ID(a3)[145,1]
ID(N2)[146,1]
LPAREN[146,3]
RPAREN[146,4]
LPAREN[147,1]
INT(63)[147,2]
INT(24)[148,10]
INT(561)[151,6]
INT(505)[152,1]
INT(7359)[153,2]
STRING("zN")[155,1]
RPAREN[155,5]
INT(237)[155,6]
INT(339)[156,1]
STRING("QjNjEA")[158,1]
ID(l)[158,9]
LPAREN[158,10]
ID(tl)[158,11]
INT(8082)[159,1]
STRING("mDXw")[160,1]
LPAREN[165,3]
INT(5244)[165,4]
ID(L)[166,1]
STRING("")[166,2]
INT(5217)[167,1]
INT(16)[168,1]
INT(187)[169,2]
LPAREN[171,6]
INT(5)[171,7]
INT(89)[173,1]
LPAREN[174,1]
INT(10)[174,2]
ID(pI)[175,1]
LPAREN[176,1]
STRING("zw")[176,2]
INT(382)[176,6]
ID(sgWum)[177,1]
RPAREN[177,6]
INT(8)[179,6]
INT(0)[180,1]
INT(7949)[181,1]
STRING("t")[182,1]
RPAREN[183,1]
INT(76)[184,1]
INT(82)[185,2]
ID(cNlj)[186,2]
STRING("Rx")[186,6]
LPAREN[186,11]
ID(r2)[186,12]
INT(621)[187,1]
STRING("Q")[188,6]
INT(523)[188,9]
INT(8515)[189,1]
ID(KGtmY5L)[192,4]
STRING("")[192,11]
INT(7)[192,13]
ID(o2)[196,5]
INT(15)[197,1]
LPAREN[198,1]
INT(721)[198,2]
RPAREN[199,1]
STRING("")[199,3]
INT(8193)[200,1]
INT(7)[201,1]
INT(0)[202,1]
RPAREN[206,1]
INT(7518)[206,2]
INT(970)[207,2]
INT(9017)[208,1]
RPAREN[209,1]
RPAREN[209,3]
LPAREN[210,1]
INT(75)[210,3]
INT(17)[217,1]
INT(0)[219,1]
INT(220)[220,2]
RPAREN[222,1]
INT(906)[223,1]
ID(DG11)[224,1]
INT(2)[225,1]
RPAREN[226,2]
ID(b32)[226,3]
INT(22)[227,1]
STRING("")[228,1]
INT(22)[231,1]
STRING("")[232,1]
ID(KTy)[232,3]
STRING("JeT")[232,6]
INT(7)[233,1]
INT(75)[234,1]
INT(1296)[238,1]
INT(372)[239,1]
LPAREN[240,1]
INT(4)[245,5]
STRING("bPv")[246,1]
STRING("dej")[246,6]
INT(44)[246,11]
STRING("THmMCx")[247,2]
INT(72)[247,10]
INT(589)[254,8]
LPAREN[255,1]
STRING("")[256,1]
LPAREN[256,3]
INT(518)[256,4]
INT(6)[257,1]
INT(353)[258,1]
INT(137)[259,1]
LPAREN[261,2]
INT(18)[261,3]
RPAREN[262,1]
INT(6385)[262,6]
LPAREN[263,1]
INT(0)[263,2]
INT(1082)[264,1]
INT(81)[265,1]
INT(89)[266,1]
ID(H)[267,1]
INT(1811)[267,7]
LPAREN[268,1]
ID(yp)[268,2]
RPAREN[268,4]
STRING("H")[268,6]
STRING("e")[269,1]
INT(9)[269,4]
LPAREN[270,1]
INT(6)[272,1]
INT(928)[273,1]
INT(389)[274,1]
RPAREN[275,1]
LPAREN[275,2]
ID(j4LT3R8)[275,3]
INT(5)[276,1]
INT(9)[277,1]
RPAREN[279,1]
INT(39)[280,1]
INT(339)[281,1]
INT(6735)[282,11]
INT(2518)[283,3]
LPAREN[284,1]
STRING("y")[284,2]
STRING("z")[284,5]
LPAREN[284,8]
INT(1348)[284,9]
STRING("X")[285,1]
INT(1692)[285,4]
ID(byAzB)[286,1]
LPAREN[286,6]
INT(9)[288,3]
LPAREN[290,1]
LPAREN[290,2]
INT(9)[290,4]
INT(21)[292,2]
ID(Y8a746)[293,1]
INT(99)[294,1]
LPAREN[295,1]
INT(9)[295,2]
INT(2)[297,1]
RPAREN[300,1]
INT(12)[300,2]
INT(5)[301,1]
INT(16)[303,1]
RPAREN[304,1]
ID(B68)[304,2]
RPAREN[305,1]
LPAREN[305,2]
STRING("")[305,3]
INT(74)[305,5]
INT(7)[306,1]
RPAREN[307,1]
INT(1530)[307,2]
INT(91)[313,14]
INT(7807)[314,1]
RPAREN[316,5]
LPAREN[316,6]
INT(8)[316,7]
INT(7535)[317,1]
RPAREN[318,1]
INT(6)[318,2]
INT(7776)[321,1]
INT(96)[323,10]
INT(906)[324,1]
INT(5)[325,1]
INT(4284)[326,1]
ID(F27)[327,2]
RPAREN[327,5]
INT(154)[327,6]
STRING("qz")[328,1]
RPAREN[328,5]
STRING("T")[328,6]
STRING("oD")[328,9]
STRING("pj")[329,1]
STRING("QMe")[329,5]
LPAREN[329,10]
ID(r2Fp63)[329,11]
STRING("")[332,1]
INT(902)[332,3]
RPAREN[333,1]
INT(3700)[333,3]
LPAREN[334,1]
LPAREN[334,2]
INT(5)[334,3]
LPAREN[335,1]
INT(97)[335,2]
RPAREN[337,1]
INT(0)[337,3]
ID(UWQ)[339,1]
STRING("")[339,5]
INT(8003)[339,7]
INT(99)[343,3]
INT(964)[344,1]
INT(5)[345,1]
STRING("eRlu")[347,1]
LPAREN[347,7]
INT(416)[347,8]
INT(8109)[348,1]
LPAREN[349,1]
STRING("")[350,1]
RPAREN[350,3]
LPAREN[350,4]
LPAREN[350,5]
RPAREN[350,6]
INT(8866)[350,7]
ID(Tpr81)[351,1]
INT(80)[353,1]
LPAREN[354,1]
INT(9493)[354,2]
INT(21)[355,1]
INT(272)[356,1]
INT(94)[358,1]
LPAREN[359,1]
INT(3)[360,1]
RPAREN[362,1]
INT(6335)[362,2]
INT(979)[364,1]
INT(6776)[365,1]
INT(8803)[366,1]
INT(5)[367,1]
RPAREN[368,1]
INT(4)[368,2]
ID(m7)[369,1]
ID(SB752)[370,1]
INT(66)[371,1]
RPAREN[376,3]
STRING("WH")[376,4]
ID(m2z)[376,8]
STRING("Uew")[376,11]
INT(3018)[376,16]
LPAREN[377,1]
ID(F6d1)[377,2]
INT(40)[378,2]
ID(I)[379,1]
STRING("yLoypLq")[379,2]
INT(849)[379,11]
ID(d8)[380,1]
INT(245)[381,1]
ID(M5966)[382,1]
ID(dP)[383,1]
RPAREN[387,19]
INT(283)[387,20]
RPAREN[388,19]
INT(69)[388,20]
RPAREN[389,1]
INT(0)[390,1]
ID(XT)[391,13]
RPAREN[391,19]
STRING("Z")[394,17]
INT(7030)[394,20]
INT(943)[396,1]
INT(35)[397,1]
INT(30)[398,1]
ID(dghxHce8DTQ58)[399,1]
INT(3)[400,1]
LPAREN[401,1]
RPAREN[401,2]
RPAREN[401,3]
ID(ywxP34)[401,5]
INT(0)[402,1]
ID(iLZ42y)[403,1]
STRING("k")[403,7]
LPAREN[403,10]
RPAREN[403,11]
LPAREN[403,12]
LPAREN[403,13]
INT(6820)[403,14]
RPAREN[405,1]
INT(6655)[405,2]
ID(C)[406,1]
INT(76)[407,1]
INT(67)[408,1]
INT(271)[409,1]
LPAREN[415,4]
ID(m)[420,3]
LPAREN[420,5]
LPAREN[422,8]
STRING("X")[423,1]
ID(I8)[423,4]
INT(6)[427,4]
RPAREN[428,2]
INT(79)[428,3]
ID(j7857)[429,1]
INT(80)[430,1]
INT(3605)[431,1]
RPAREN[432,1]
INT(92)[432,2]
INT(9574)[433,1]
INT(5)[434,1]
INT(7363)[435,1]
RPAREN[437,7]
ID(Kg)[437,8]
INT(747)[438,1]
INT(9358)[440,2]
RPAREN[441,1]
INT(0)[443,1]
LPAREN[444,1]
STRING("")[444,2]
INT(73)[444,4]
RPAREN[446,1]
INT(399)[446,2]
RPAREN[447,1]
STRING("Iy")[447,2]
INT(2851)[447,6]
RPAREN[449,1]
INT(2)[449,2]
ID(E6)[450,1]
INT(9)[451,1]
INT(8106)[452,1]
INT(25)[453,1]
INT(17)[454,2]
INT(7455)[455,1]
RPAREN[456,1]
RPAREN[461,1]
STRING("")[461,2]
RPAREN[461,4]
INT(56)[461,5]
INT(790)[462,1]
ID(p10)[463,1]
STRING("J")[463,4]
INT(761)[463,7]
ID(h9Z007)[464,1]
STRING("hFLifIAT")[471,6]
INT(356)[471,16]
INT(994)[472,1]
INT(0)[473,1]
INT(8)[475,1]
INT(1554)[477,1]
LPAREN[478,1]
LPAREN[478,7]
RPAREN[478,8]
INT(276)[478,9]
ID(fY67)[479,1]
LPAREN[480,1]
INT(0)[480,2]
INT(98)[481,1]
RPAREN[482,1]
LPAREN[482,2]
INT(588)[482,3]
ID(s9)[486,1]
RPAREN[486,3]
INT(317)[487,1]
STRING("tmHV")[489,1]
INT(8)[491,1]
RPAREN[493,1]
INT(9)[493,2]
RPAREN[494,1]
INT(2010)[494,2]
INT(848)[495,1]
INT(9)[496,1]
INT(4)[497,1]
STRING("YAVYT")[498,1]
LPAREN[498,8]
INT(35)[500,1]
ID(AU)[501,1]
ID(qv0J)[501,11]
INT(548)[502,1]
INT(7)[509,13]
ID(JOEm9J)[510,1]
RPAREN[519,5]
INT(1237)[520,1]
ID(J)[521,1]
ID(yYU9435)[522,1]
INT(83)[523,1]
ID(B)[525,3]
STRING("")[525,4]
RPAREN[525,6]
INT(7579)[525,7]
STRING("j")[527,1]
LPAREN[527,4]
INT(8)[527,5]
RPAREN[528,1]
INT(737)[528,2]
STRING("CXUDcI")[530,1]
INT(3)[530,9]
STRING("S")[531,1]
INT(10)[531,4]
RPAREN[532,1]
STRING("")[532,2]
INT(815)[532,4]
RPAREN[536,1]
INT(87)[536,2]
LPAREN[537,1]
INT(13)[537,2]
INT(29)[538,1]
INT(844)[539,1]
INT(684)[542,1]
INT(4)[543,1]
INT(1)[544,1]
ID(Q)[545,1]
ID(G7106)[545,3]
INT(3941)[546,1]
INT(383)[548,1]
STRING("K")[551,1]
INT(9215)[551,4]
RPAREN[552,1]
INT(0)[552,2]
ID(B)[553,1]
STRING("")[553,8]
LPAREN[554,1]
ID(jwx)[554,2]
LPAREN[554,5]
LPAREN[554,6]
LPAREN[554,7]
LPAREN[556,5]
RPAREN[557,7]
RPAREN[557,8]
LPAREN[557,9]
STRING("iQ")[560,1]
ID(N8D7392)[560,5]
ID(t6l9Z)[561,1]
RPAREN[569,5]
ID(rcrmEZK32822)[569,6]
INT(9210)[570,1]
INT(7218)[572,1]
LPAREN[573,1]
INT(37)[573,3]
INT(1)[574,2]
LPAREN[575,1]
INT(46)[575,2]
RPAREN[576,1]
INT(2)[578,1]
ID(z)[579,1]
RPAREN[579,2]
LPAREN[579,3]
INT(77)[580,1]
INT(3242)[583,1]
LPAREN[584,1]
LPAREN[585,1]
ID(Gn9M)[585,2]
LPAREN[587,5]
LPAREN[588,1]
INT(103)[593,4]
INT(1599)[595,1]
INT(15)[597,3]
LPAREN[598,1]
STRING("d")[601,1]
INT(7682)[601,4]
INT(5057)[603,1]
STRING("")[604,1]
INT(2)[608,5]
INT(505)[609,1]
INT(837)[610,1]
INT(326)[611,1]
INT(0)[612,1]
INT(53)[615,1]
RPAREN[616,1]
INT(24)[616,2]
INT(9631)[617,1]
LPAREN[618,1]
RPAREN[618,2]
INT(5)[619,1]
INT(9931)[620,1]
RPAREN[621,1]
STRING("")[621,2]
LPAREN[621,4]
RPAREN[621,5]
STRING("")[621,6]
INT(262)[622,1]
ID(DE729S3)[623,1]
LPAREN[623,8]
INT(454)[626,14]
ID(Ww2e)[627,1]
RPAREN[627,5]